tsconfig.json

```diff
@@ -15,7 +15,10 @@
       "@ui/networkData/*": ["ui/networkData/*"],
       "@ui/styles/*": ["ui/styles/*"],
       "@ui/images/*": ["ui/public/images/*"],
-      "@ui/icons/*": ["ui/public/icons/*"]
+      "@ui/icons/*": ["ui/public/icons/*"],
+      "@chains/polygon/*": ["chains/src/polygon/*"],
+      "@chains/bsc/*": ["chains/src/bsc/*"],
+      "@chains/moonbeam/*": ["chains/src/moonbeam/*"]
```

Comment:
should we also add the testnet chains?

Reply:
i've added chapel. I guess we might never need these though. But let's see, can't hurt.